### PR TITLE
CIT: add test docs

### PR DESCRIPTION
**Pull request by hopkiw**
document each test

### CODE REVIEW

File: imagetest/README.md [hunk 1]
```diff
@@ -79,3 +79,13 @@ It is suggested to start by copying an existing test package.
 From the root of this repository:
 
     $ docker build -t cloud-image-tests -f imagetest/Dockerfile .
+
+## What is being tested ##
+
+The tests are a combination of various types - end to end tests on certain
+software components, image validations and feature validations, etc. The
+collective whole represents the quality assurance bar for releasing a [supported
```
Comment (zmarano) [inline imagetest/README.md:87]: We need to clarify this. (and link to somewhere)
Obviously, we can only say this against the images that Google produces. We do list those here: https://cloud.google.com/compute/docs/images/os-details under the Built by section

File: imagetest/test_suites/README.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
+# Image test suites #
+
+## What is being tested ##
+
+The tests are a combination of various types - end to end tests on certain
+software components, image validations and feature validations, etc. The
+collective whole represents the quality assurance bar for releasing a [supported
```
Comment (zmarano) [inline imagetest/test_suites/README.md:7]: ditto

File: imagetest/test_suites/README.md [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+### Test suite: disk ###
+
+#### TestDiskResize ####
+Validate the filesystem is resized on reboot after a disk resize.
+
+Background
+
+A convenience feature offered on supported GCE Images, if you resize the
+underlying disk to be larger, then a set of scripts invoked during boot will
+automatically resize the root partition to take advantage of the new space.
```
Comment (zmarano) [inline imagetest/test_suites/README.md:24]: root partition and filesystem

File: imagetest/test_suites/README.md [hunk 4]
```diff
@@ -0,0 +1,293 @@
...
+'forever')
+
+Background
+
+We guarantee shutdown scripts will block the system shutdown process until the
+script completes. For scripts which never complete, this would cause the server
+to remain in a 'shutting down' state forever. However, VMs that are stopped via
+the API are first sent an ACPI soft-shutdown signal which triggers the OS
+shutdown process, invoking this script. But after a set amount of time
+(currently 2 minutes), if the VM is still running, the GCE API will hard-reset
```
Comment (zmarano) [inline imagetest/test_suites/README.md:80]: I believe its 90 seconds per https://cloud.google.com/compute/docs/shutdownscript

File: imagetest/README.md (hunk 1 above)
Comment (hopkiw) [reply imagetest/README.md:87]: sorry, what should we clarify?

File: imagetest/test_suites/README.md (hunk 2 above)
Comment (hopkiw) [reply imagetest/test_suites/README.md:7]: ditty to the 'we need to clarify this' comment? you've selected the same three lines here

File: imagetest/README.md (hunk 1 above)
Comment (zmarano) [reply imagetest/README.md:87]: Oh- the link... [supported GCE Images][gce-images] is not a thing.

**Pull request merged**